bot.py

```diff
@@ -8,102 +8,30 @@
 import discord
 from discord.ext import commands
 
+
 BOT_PREFIX = os.environ['discord_prefix']
 TOKEN = os.environ['discord_token']
 WEATHER_API_KEY = os.environ['open_weather_map_key']
 NASA_API_KEY = os.environ['nasa_api_key']
```

Comment:
These environment variables could probably be defined in the function files for the commands.